ecmd-core/ext/fapi2/capi/target_types.H

```diff
@@ -161,11 +161,22 @@ enum TargetFilter : uint64_t;
 // attributeOverride tool requires x86.nfp compile
 #ifndef CONTEXT_x86_nfp
 /// @cond
-constexpr TargetType operator|(TargetType x, TargetType y)
+    constexpr TargetType operator|(TargetType x, TargetType y)
```

Comment:
remove extra whitespace?

Reply:
this was how the ekb formatted it

Reply:
The ekb code prettifier insists on this whitespace and puts it back when I remove it, so who am I to disagree? ;)